app/models/team.rb

```diff
@@ -2,6 +2,8 @@ class Team < ApplicationRecord
   has_many :students
   validates :program, :number, :name, presence: true, allow_blank: false
 
```

Comment:
Can we put all the validations together.

```
has_many :students

validates :program, :number, blah blah
validates :program, inclusion: blah blah
```

Reply:
:+1: